### PR TITLE
docs: fix missing colon, attempt to fix the code-block in channel.purge

### PR DESCRIPTION
**Pull request by sillyElvy**
## About

This pull request is about (X), which does (Y).

## Checklist

- [ ] I've ran `pre-commit` to format and lint the change(s) made.
- [ ] I've checked to make sure the change(s) work on `3.8.6` and higher.
- [ ] This fixes/solves an [Issue](https://github.com/goverfl0w/discord-interactions/issues).
  - (If existent):
- [ ] I've made this pull request for/as: (check all that apply)
  - [x] Documentation
  - [ ] Breaking change
  - [ ] New feature/enhancement
  - [ ] Bugfix

### CODE REVIEW

File: interactions/api/models/channel.py [hunk 1]
```diff
@@ -724,10 +724,14 @@ async def purge(
 
         .. warning:: Calling this method can lead to rate-limits when purging higher amounts of messages.
 
+
```
Comment (Toricane) [inline interactions/api/models/channel.py]: I think this new line is unnecessary when compared to the other docstrings with a code block.

File: interactions/api/models/channel.py [hunk 2]
```diff
@@ -724,10 +724,14 @@ async def purge(
...
         .. code-block:: python
+
             def check_pinned(message):
                 return not message.pinned  # This returns `True` only if the message is the message is not pinned
             await channel.purge(100, check=check_pinned)  # This will delete the newest 100 messages that are not pinned in that channel
+
+
```
Comment (Toricane) [inline interactions/api/models/channel.py]: One new line should be sufficient here.

**Review by Toricane — CHANGES_REQUESTED** [review]
A couple of things

**Comment by sillyElvy** [comment]
> A couple of things

Well let's see how the rtd build turns out now

**Review by Toricane — APPROVED** [review]
LGTM!

**Comment by Toricane** [comment]
> > A couple of things
> 
> Well let's see how the rtd build turns out now

OK cool it works <https://interactionspy.readthedocs.io/en/unstable/api.models.channel.html>

**Pull request merged**